packages/fluentui/react-northstar/src/themes/teams/staticStyles/globalStyles.ts

```diff
@@ -17,4 +17,8 @@ export const globalStyles: StaticStyleFunction = siteVars => ({
   '*:after': {
     boxSizing: 'border-box',
   },
+  /* Adding priority for HTML `hidden` attribute to be applied correctly */
```

Comment:
Based on the discussions here https://css-tricks.com/the-hidden-attribute-is-visibly-weak/ and here https://meowni.ca/hidden.is.a.lie.html - I'm opting to add !important to the `[hidden]` style as the most robust solution